js-api-doc/value/color.d.ts

```diff
@@ -10,119 +89,467 @@ import {Value} from './index';
+      [key in ChannelNameHsl]?: number | null;
+    } & {
+      space?: ColorSpaceHsl;
+    }
+  ): SassColor;
 
   /**
    * Changes one or more of this color's HWB channels and returns the result.
+   *
+   * @throws `Error` if `space` is missing and this color is not in a legacy
+   * color space (`rgb`, `hsl`, or `hwb`).
+   * @throws `Error` if `alpha` is set and isn't `null` or a number between `0`
+   * and `1`.
+   */
+  change(
+    options: {
+      [key in ChannelNameHwb]?: number | null;
+    } & {
+      space?: ColorSpaceHwb;
+    }
+  ): SassColor;
+
+  /**
+   * Changes one or more of this color's Lab channels and returns the result.
+   *
+   * @throws `Error` if `space` is missing and this color is not in the Lab or
+   * Oklab color spaces.
+   * @throws `Error` if `alpha` is set and isn't `null` or a number between `0`
+   * and `1`.
+   * @throws `Error` if `lightness` is set and isn't `null` or a number between
+   * `0` and the maximum channel value for `space` (or this color's color space,
+   * if `space` is missing)—`100` for Lab or `1` for Oklab.
+   */
+  change(
+    options: {
+      [key in ChannelNameLab]?: number | null;
+    } & {
+      space?: ColorSpaceLab;
+    }
+  ): SassColor;
+
+  /**
+   * Changes one or more of this color's LCH channels and returns the result.
+   *
+   * @throws `Error` if `space` is missing and this color is not in the LCH or
+   * Oklch color spaces.
+   * @throws `Error` if `alpha` is set and isn't `null` or a number between `0`
+   * and `1`.
+   * @throws `Error` if `lightness` is set and isn't `null` or a number between
+   * `0` and the maximum channel value for `space` (or this color's color space,
+   * if `space` is missing)—`100` for LCH or `1` for Oklch.
+   */
+  change(
+    options: {
+      [key in ChannelNameLch]?: number | null;
+    } & {
+      space?: ColorSpaceLch;
+    }
+  ): SassColor;
+
+  /**
+   * Changes one or more of this color's RGB channels and returns the result.
+   *
+   * @throws `Error` if `space` is missing and this color is not in a legacy
+   * color space (`rgb`, `hsl`, or `hwb`).
+   * @throws `Error` if `alpha` is set and isn't `null` or a number between `0`
+   * and `1`.
+   */
+  change(
+    options: {
+      [key in ChannelNameRgb]?: number | null;
+    } & {
+      space?: ColorSpaceRgb;
+    }
+  ): SassColor;
+
+  /**
+   * Changes one or more of this color's XYZ channels and returns the result.
+   *
+   * @throws `Error` if `space` is missing and this color is not in an XYZ color
+   * space.
+   * @throws `Error` if `alpha` is set and isn't `null` or a number between `0`
+   * and `1`.
    */
-  change(options: {
-    hue?: number;
-    whiteness?: number;
-    blackness?: number;
-    alpha?: number;
-  }): SassColor;
+  change(
+    options: {
+      [key in ChannelNameXyz]?: number | null;
+    } & {
+      space?: ColorSpaceXyz;
+    }
+  ): SassColor;
+
+  /**
+   * This color's red channel, between `0` and `255`.
```

Comment:
Mention that this is specifically in the RGB color space. Also below.

Reply:
Addressed in https://github.com/sass/sass/pull/3718/commits/4efd448207eb7e86024a985a89422c87766133ea